docs/usage/configuration/config_documentation.md

```diff
@@ -3014,6 +3014,12 @@ Options for each entry include:
      which is set to the claims returned by the UserInfo Endpoint and/or
      in the ID Token.
 
+* `backchannel_logout_enabled`: set to true to process OIDC Back-Channel Logout notifications. 
+  Those notifications are expected to be received on `/_synapse/client/oidc/backchannel_logout`.
```

Comment:
What does this default to?

Reply:
The default is `false`, but I assumed that the wording ("set to true") was enough to suggest that the default is `false`